lms/djangoapps/teams/static/teams/js/views/edit_team.js

```diff
@@ -215,7 +213,7 @@
                         url = 'teams/' + this.topicID + '/' + this.teamModel.get('id');
                     }
                     Backbone.history.navigate(url, {trigger: true});
-                }
+                },
```

Comment:
I'll come back through and reset this file to how it was, none of the changes here are relevant to the instructor toolbar.